common/daemon.h

```diff
@@ -124,30 +119,24 @@ class CF_logger
             char time_buffer[80];
             strftime( time_buffer, 80, "%Y-%m-%d %H:%M:%S", local_time );
             fprintf( fp, "%s: ", time_buffer );
-            fprintf( fp, format, buffer );
+            fprintf( fp, format, message );
             fprintf( fp, "\n" );
             fclose( fp );
         }
-        std::string log_buf = std::string( buffer, strlen( buffer ) );
+        std::string log_buf = std::string( message );
         log_ring_buffer[log_buffer_count] = log_buf;
         log_buffer_count = ( log_buffer_count + 1 ) % MAX_LOG_BUFFER_COUNT;
         close( fd );
     }
 
-    template <typename... Logs> void logger( const int level, const char* fmt, Logs... logs )
+    void logger( const int level, const char* fmt, const char* logs )
```

Comment:
We can remove the formatting string fmt since variadic prints are going away

Reply:
Do you mean sd_journal_print is going away and can be removed from here?

Reply:
fixed